components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -201,7 +201,7 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
                 <div className='SidebarCategory_newDropBox'>
                     <div
                         ref={this.newDropBoxRef}
-                        className={classNames('SidebarCategory_newDropBox-content', {
+                        className={classNames('SidebarCategory_newDropBox-content animating', {
```

Comment:
`animating` as a verb implies that something is currently animating ... wonder if we should change to `animates` or `animated` to indicate that the element can/will animate ... or maybe no additional class is really needed at all?

If you do actually want to use `animating` as the class name, maybe only add it when the animation needs to happen and then remove once done as a number of popular libraries do for this kind of thing.

Reply:
the `animating` was included even before I did my changes. I simple added it .. Although I just checked (and removed the hard coded class): It is actually added and removed automatically and I just did not notice that beforehand. 

Thanks for pointing that out! 👍🏼

Reply:
Ah, lol, sounds good! 😄